### DISCUSSION TITLE
[React] Tooltip on buttons on iOS

### DISCUSSION THREAD

**Discussion by TheShrike** [floating-ui/floating-ui#2716] [React]
On iOS you can't focus a button, it's therefore not possible to have the tooltip displayed on a button, without the button click event being executed

This is from your React DOM Interactions documentation example, I just added an alert on the onclick event:
https://codesandbox.io/s/nervous-wind-je0ytp

Is it possible to show the tooltip without having the alert appear? is this a bug? is a workaround needed?

On android you can trigger the focus event by holding for a long time, so the problem does not occur.

**Comment by atomiks** [comment]
> On android you can trigger the focus event by holding for a long time

To make it work on iOS we probably need to handle touch events and check for long presses. Maybe it should be a dedicated hook or a new prop for `useHover` or such.

**Comment by FezVrasta** [comment]
Could the pointer events help?
